ozaria/site/views/play/level/LevelGoals.vue

```diff
@@ -1,33 +1,31 @@
-<template lang="pug">
-  div
-    // TODO: Split this into two components, one the ul, the other the goals-status
-    ul#primary-goals-list(dir="auto")
-      level-goal(
-        v-for="goal in levelGoals",
-        :goal="goal",
-        :state="goalStates[goal.id]",
-      )
-    level-goal(
-      v-if="conceptGoals.length",
-      :goal="{ name: $t('play_level.use_at_least_one_concept') }",
-      :state="{ status: conceptStatus }",
-    )
-    ul#concept-goals-list(dir="auto" v-if="conceptGoals.length")
-      level-goal.concept-goal(
-        v-for="goal in conceptGoals",
-        :goal="goal",
-        :state="goalStates[goal.id]",
-      )
-      
-    div.goals-status.rtl-allowed(v-if="showStatus")
-      span {{ $t("play_level.goals") }}
-      span.spr :
-      span(v-if="classToShow === 'running'").goal-status.running {{ $t("play_level.running") }}
-      span(v-if="classToShow === 'success'").goal-status.success {{ $t("play_level.success") }}
-      span(v-if="classToShow === 'incomplete'").goal-status.incomplete {{ $t("play_level.incomplete") }}
-      span(v-if="classToShow === 'timed-out'").goal-status.timed-out {{ $t("play_level.timed_out") }}
-      span(v-if="classToShow === 'failing'").goal-status.failure {{ $t("play_level.failing") }}
-
+<template>
+  <div>
+    <div v-if="showStatus" class="goals-status rtl-allowed">
+      <span>{{ $t("play_level.goals") }} : {{ $t("play_level." + goalStatus) }}</span>
+    </div>
+    <div class="level-goals">
+      <!-- TODO: Split this into two components, one the ul, the other the goals-status-->
+      <ul id="primary-goals-list" dir="auto">
+        <level-goal v-for="goal in levelGoals"
+                    :key="goal.id"
+                    :goal="goal"
+                    :state="goalStates[goal.id]">
+        </level-goal>
+      </ul>
+      <level-goal v-if="conceptGoals.length"
```

Comment:
Unsure how to test the conceptGoals. I was able to manually test the levelGoals.

Why is the conceptGoals split between two level-goals?

Reply:
This was kept to be compatible with the way goals were being handled in the old code. In case concept goals did end up being used in some of the content, I wanted it to be supported at least. Perhaps it is better to cut it from the code and add it back in if that situation ever arises.